tests/plugins/test_tag_from_config.py

```diff
@@ -108,8 +108,8 @@ def test_tag_from_config_plugin_generated(tmpdir, docker_tasker, tags, name,
 
 
 @pytest.mark.parametrize(('inspect', 'error'), [  # noqa
-    ({'Labels': {}}, "KeyError('name'"),
-    ({}, "KeyError('Labels'"),
+    ({'Labels': {}}, "KeyError(<object"),
+    ({}, "KeyError('Labels"),
```

Comment:
Not sure I understand why either of these changes are needed.

Reply:
Second one is a typo.
First one is because we're no longer looking up 'name', we're looking up LABEL_TYPE_NAME, which is an object. When it isn't found, the KeyError is against the object.